server/controllers/LibraryController.js

```diff
@@ -218,30 +252,55 @@ class LibraryController {
           }
         })
       }
-      libraryItems = naturalSort(libraryItems).by(sortArray)
     }
 
-    if (payload.collapseseries) {
-      libraryItems = libraryHelpers.collapseBookSeries(libraryItems, this.db.series)
-      payload.total = libraryItems.length
-    } else if (filterSeries) {
-      // Book media when filtering series will include series object on media metadata
-      libraryItems = libraryItems.map(li => {
-        var series = li.media.metadata.getSeries(filterSeries)
-        var liJson = payload.minified ? li.toJSONMinified() : li.toJSON()
-        liJson.media.metadata.series = series
-        return liJson
-      })
-      libraryItems = naturalSort(libraryItems).asc(li => li.media.metadata.series.sequence)
-    } else {
-      libraryItems = libraryItems.map(li => payload.minified ? li.toJSONMinified() : li.toJSON())
+    if (sortArray.length) {
+      libraryItems = naturalSort(libraryItems).by(sortArray)
     }
 
-    if (payload.limit) {
-      var startIndex = payload.page * payload.limit
-      libraryItems = libraryItems.slice(startIndex, startIndex + payload.limit)
-    }
```

Comment:
Is there a reason this was removed here? Just got notified on Discord about an issue and saw the limit is not applied.

Reply:
No, that was an accident. My guess is that this happened the first time I rebased the patch, where we had both made changes to this file, and I just made a mistake. Sorry.